common/modal.js

```diff
@@ -145,6 +146,27 @@
             }
         }
 
+        // NOTE: only used for DifferentUserConflictError
+        if (exception.showContinueBtn) {
```

Comment:
Why are you not using the same pattern of `expection instanceof DifferentUserConflictError` that we have in the lines above?  (I'm not sure which is better/cleaner and I'm just asking).

Reply:
I could now. Before my changes included me relying on the else case in the above if/else chain to show the reload button and append the text for it. But since we decided to use different text for reload in this case, that has changed.

Reply:
I don't know if what I'm seeing is related to this or not. But in recordedit I'm seeing the "reload" message twice.

![Screen Shot 2020-04-06 at 11 43 32 AM](https://user-images.githubusercontent.com/2129750/78593677-ddce4180-77fb-11ea-91d3-7573bc87f26a.png)

Reply:
I'm not sure why this happened for you. Since changing the code to check the exception type, the latter reload message should never be appended because the else case won't be reached.

Reply:
I'm not sure what changed but I cannot reproduce this anymore. I'll mark this as "resolved".